Sources/WWDCNotes/WWDCNotes.docc/WWDC24/WWDC24-10124-Whats-new-in-AppKit.md

```diff
@@ -8,11 +8,107 @@ Discover the latest advances in Mac app development. Get an overview of the new
    @CallToAction(url: "https://developer.apple.com/wwdc24/10124", purpose: link, label: "Watch Video (19 min)")
 
    @Contributors {
-      @GitHubUser(<replace this with your GitHub handle>)
+      @GitHubUser(philptr)
    }
 }
 
-😱 "No Overview Available!"
+# New macOS Features
 
-Be the hero to change that by watching the video and providing notes! It's super easy:
- [Learn More…](https://wwdcnotes.github.io/WWDCNotes/documentation/wwdcnotes/contributing)
+## Writing Tools
+- Apps get the features automatically, but can customize behavior using new APIs
+- Watch [Get started with Writing Tools](https://developer.apple.com/wwdc24/10168)
+
+## Genmoji
+- Genmoji are images, so some adoption may be needed
+- Watch [Bring expression to your app with Genmoji](https://developer.apple.com/wwdc24/10220)
+
+## Image Playground
+- Instantiate `ImagePlaygroundViewController` and assign its delegate
+- Can customize the behavior by specifying initial `concepts` and `sourceImage`
+- Users can still choose different images and concepts as part of the experience
+- Get the generated image from the app's sandboxed temporary directory
+- Consider adding Image Playground as an image source
+
+## Window Tiling
+- Makes it *very* fast to move windows into some common arrangements
+- Holding `Option` while dragging shows a preview of the window immediately
+- Available in the Window > Move & Resize menu, and accessible using keyboard shortcuts
+- Can resize windows at the same time while they are side by side
+- Can select pre-built window arrangements from the Window menu
+
+@Image(source: "window-tiling.png", alt: "Tiled windows can be resized simultaneously.")
```

Comment:
There's no need to add the `.png` extension. But it's also not wrong, so it's fine. 👍
But I would prefer if images were in JPEG format as that saves space in the repo, on server, and for readers.
I'll do it after the merge this time, but in future sessions please convert to JPEG right away. See the [docs]()https://wwdcnotes.github.io/WWDCNotes/documentation/wwdcnotes/contributing/#Image-Name--Format)